apigee/src/main/apigee/apiproxies/apigee-saml-idp/apiproxy/resources/properties/settings.properties

```diff
@@ -0,0 +1,16 @@
+# settings for this proxy.
```

Comment:
Can you update the comments in this file if they're no longer relevant? Also, are these all OK to expose publicly?